variables.tf

```diff
@@ -38,4 +38,5 @@ variable "retention_days" {
 variable "gitlab_snapshot_id" {
     type    = string
     default = null
-}
\ No newline at end of file
+}
+variable "swap_volume_size" {}
```

Comment:
please add a variable type and a default value

Reply:
Done, already added.

Reply:
Why don't you add a default number value to don't force users to set one?

Reply:
Done, put 8 as a default value